BankAccountApp/BankAccountApp/BankAccount.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BankAccountApp
+{
+    class BankAccount : GeneralAccount, IAccountOperations
+    {
+        public enum ACCOUNT_STATUS { OPEN, CLOSED };
+
+        private decimal balance;
+        private Owner owner;
+        private ACCOUNT_STATUS accountStatus;
+        private List<Operation> listaOperatii;
+
+        public List<Operation> ListaOperatii
+        {
+            get { return listaOperatii; }
+            private set { listaOperatii = value; }
+        }
+
+
+        public ACCOUNT_STATUS AccountStatus
+        {
+            get { return accountStatus; }
+            set { accountStatus = value; }
+        }
+
+
+        public Owner Owner
+        {
+            get { return owner; }
+            set { owner = value; }
+        }
+
+        public decimal Balance
+        {
+            get { return balance; }
+            protected set { balance = value; }
+        }
+
+        public BankAccount(string accountID, Owner owner, DateTime creationDate) : base(accountID, creationDate)
```

Comment:
creationDate, eu as vedea sa nu fie necesar in constructor, ci sa se asigneze automat atunci cand se instantiaza clasa.

Reply:
De acord. Voi face modificarea.